src/index.html

```diff
@@ -6,9 +6,40 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Henny+Penny&family=Lexend+Deca:wght@400;500;600;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="home-image-link">
+        <img
+        src="../src/images/logo.png"
+        alt="moyo logo"
+        class="logo"
+        >
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav-item "><a href="#" class="nav_tag is-active" >apple</a></li>
```

Comment:
Please do this kind of indentation all over the code
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">Apple</a>
          </li>
```